### PR TITLE
Added new playground templates

### PR DESCRIPTION
**Pull request by PirateJC**
Added playground templates for:
creating a point light
creating a spot light
creating a directional light
creating a hemispheric light
showing the inspector
Setting up a shadow generator
Loading a node material from a snippet

### CODE REVIEW

File: Playground/templates.json [hunk 1]
```diff
@@ -23,6 +23,42 @@
     "insertText" : "var ground = BABYLON.MeshBuilder.CreateGround(\"${1:ground}\", {width: ${2:6}, height: ${3:6}}, scene);",
     "language" : "javascript"
   },
+  {
+    "label" : "Create a point light",
+    "documentation" : "https://doc.babylonjs.com/babylon101/lights#the-point-light",
+    "insertText" : "var pointLight = new BABYLON.PointLight(\"${1:pointLight}\", ${2:new BABYLON.Vector3(0,5,0)}, scene);",
```
Comment (deltakosh) [inline Playground/templates.json:29]: maybe we could add tab over x,y,z for the position

File: Playground/templates.json [hunk 2]
```diff
@@ -23,6 +23,42 @@
...
+  {
+    "label" : "Create a point light",
+    "documentation" : "https://doc.babylonjs.com/babylon101/lights#the-point-light",
+    "insertText" : "var pointLight = new BABYLON.PointLight(\"${1:pointLight}\", ${2:new BABYLON.Vector3(0,5,0)}, scene);",
+    "language" : "javascript"
+  },
+  {
+    "label" : "Create a directional light",
+    "documentation" : "https://doc.babylonjs.com/babylon101/lights#the-directional-light",
+    "insertText" : "var dirLight = new BABYLON.DirectionalLight(\"${1:dirLight}\", ${2:new BABYLON.Vector3(-0.5,-1,-0.5)}, scene);",
```
Comment (deltakosh) [inline Playground/templates.json:35]: same here maybe? not sure, open to discussion

File: Playground/templates.json [hunk 3]
```diff
@@ -40,5 +76,11 @@
   "documentation" : "https://doc.babylonjs.com/resources/external_pg_assets",
   "insertText" : "BABYLON.SceneLoader.ImportMesh(\"${1:meshName}\", \"${2:url to the mesh parent directory}\", \"${3:Mesh filename.fileextension}\", scene, function(newMeshes){\n\n});",
   "language" : "javascript"
+  },
+  {
+    "label" : "Setup a shadow generator",
+    "documentation" : "https://doc.babylonjs.com/babylon101/shadows",
+    "insertText" : "var shadowGenerator = new BABYLON.ShadowGenerator(${1:size}, ${2:the pointlight source});\nshadowGenerator.getShadowMap().renderList.push(${3:the mesh that casts a shadow});\n${4:mesh that receives the shadow}.receiveShadows = true;",
```
Comment (deltakosh) [inline Playground/templates.json:83]: not just point light, all lights

File: Playground/templates.json (hunk 2 above)
Comment (PirateJC) [reply Playground/templates.json]: Yeah this is a fantastic idea. Updated in the latest

File: Playground/templates.json (hunk 1 above)
Comment (PirateJC) [reply Playground/templates.json]: Yeah this is a fantastic idea. Updated in the latest

**Pull request merged**